packages/discovery-react-components/src/components/SearchResults/components/Result/__tests__/Result.test.tsx

```diff
@@ -3,53 +3,62 @@ import React from 'react';
 import { render, fireEvent, RenderResult } from '@testing-library/react';
```

Comment:
I started making these changes inside of `Result2.test.tsx` instead of this file